sim/SIM-PROTOCOL.md

```diff
@@ -39,7 +39,7 @@ The beginning of a battle will look something like this:
     |
     |start
 
-`|player|PLAYER|USERNAME|AVATAR`
+`|player|PLAYER|USERNAME|AVATAR|ELO`
```

Comment:
I'm not sure if `|player|` is the correct message type to add it, though. Note that `|player|` is also sent every time a player goes online/offline or is substituted in. It's not the sort of message type for specific information intended to be sent only once at the beginning of a battle.

Reply:
The only other option is a new`|rating|username|elo`. Or maybe a `|battleinfo|JSON` for data that are absolute (like ratings in this case)?

Reply:
I wonder if we already have a `rating` message type. `battle.js` _did_ start life as a PO log viewer, after all.

Reply:
Looks like no.

I'm not horribly against the rating being here. As long as it's optional. I don't really want it to be sent for every reconnect or something.

Reply:
It's sent as `0` on everything that's not `addPlayer`, so the client will ignore it.

Reply:
Or are you worried about bandwidth?

Reply:
I'm worried about readability, partially. It should be sent blank instead of as `0`.

Reply:
This is done here https://github.com/Zarel/Pokemon-Showdown/pull/5595/commits/247f1de42d80d3a6cdaf310dab589185bfd25cd5.